server/routerlicious/packages/lambdas/src/nexus/connect.ts

```diff
@@ -92,6 +93,9 @@ function composeConnectedMessage(
 		initialMessages: [],
 		initialSignals: [],
 		supportedVersions: ProtocolVersions,
+		supportedFeatures: {
+			submit_signals_v2: true,
+		},
```

Comment:
It looks like this gets deployed to FRS without any further local server staging? @znewton I will approve this PR after all test changes requests are incorporated, but will ping you for FRS approval before we merge it.